project1/styles.css

```
@@ -0,0 +1,113 @@
+/* for my pullquote */
+
+blockquote {
+    margin: 0;
+}
+
+/* for my pullquote */
+
+blockquote p {
+    font-family: 'Satisfy', cursive;
+    font-size: 40px;
+    line-height: 1.25;
+    text-align: center;
+}
+
+/* for my pullquote */
+
+blockquote {
+    margin-left: 150px;
+    margin-right: 150px;
+}
+
+/* I wanted the background color of the <body> to be the same as the background color of the <main> element */
+
+body {
+    background-color: rgb(14, 190, 204);
+    color: whitesmoke;
+    margin: 125px;
+}
+
+
+/* for adding style to the bullets between my sections */
+
+div {
+    font-size: 18px;
+    letter-spacing: 6px;
+    line-height: 1;
+    text-align: center;
+}
+
+
+header {
+    color: navy;
+    font-family: 'Denk One', sans-serif;
+    font-size: 50px;
+    font-weight: 700px;
```

`font-weight` doesn't use any units; it's either a keyword (like `normal` or `bold`) or a number 100-900 without any units, like so:

```suggestion
    font-weight: 700;
```